pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -73,31 +73,11 @@ macro_rules! assert_add_claim {
     };
 }
 
-fn make_ticker_env(owner: AccountKeyring, token_name: AssetName) -> (Ticker, IdentityId) {
-    let owner_id = register_keyring_account(owner.clone()).unwrap();
-
-    // 1. Create a token.
-    let token = SecurityToken {
-        name: token_name,
-        owner_did: owner_id,
-        total_supply: 1_000_000,
-        divisible: true,
-        ..Default::default()
-    };
-
-    let ticker = Ticker::try_from(token.name.0.as_slice()).unwrap();
-    assert_ok!(Asset::base_create_asset_and_mint(
-        Origin::signed(owner.public()),
-        token.name.clone(),
-        ticker,
-        token.total_supply,
-        true,
-        token.asset_type.clone(),
-        vec![],
-        None,
-    ));
-
-    (ticker, owner_id)
+fn create_token() -> (Ticker, SecurityToken<u128>) {
```

Comment:
Looks like this was copy-pasta-ed from the portfolio tests... could you dedup with that?